src/neptune_tensorflow_keras/impl/__init__.py

```diff
@@ -39,21 +40,13 @@
             pip install tensorflow"""
         raise ModuleNotFoundError(msg) from exc
 
-try:
-    # neptune-client=0.9.0+ package structure
-    from neptune.new import Run
-    from neptune.new.exceptions import NeptuneException
-    from neptune.new.integrations.utils import (
-        expect_not_an_experiment,
-        verify_type,
-    )
-    from neptune.new.types import File
-except ImportError:
-    # neptune-client>=1.0.0 package structure
```

Comment:
Wouldn't it make sense to keep this (given that this is for v1.0)

Reply:
It won't work because we are using `append` that didn't exist before.